src/Components/Facility/LocationManagement.tsx

```diff
@@ -287,11 +287,9 @@ const Location = ({
         id="manage-bed-button"
         variant="secondary"
         border
-        className={cn(
-          "mt-3 flex w-full items-center justify-between",
-          totalBeds != null && "opacity-50",
-        )}
-        href={`location/${id}/beds`}
+        className={cn("mt-3 flex w-full items-center justify-between")}
+        href={totalBeds != null ? `location/${id}/beds` : undefined}
```

Comment:
Intresting, why are we having a button when its a link 🤔

Reply:
@bodhish the button component does wrap it in a `Link` component if `href` is present


<img width="1094" alt="image" src="https://github.com/user-attachments/assets/e5823f70-5d2f-4538-8ad8-3663774ebac2">